dist/index.js

```diff
@@ -2760,7 +2760,7 @@ module.exports = require("crypto");
 
 "use strict";
 
-// Older verions of Node.js might not have `util.getSystemErrorName()`.
```

Comment:
For my reference, this appears to originate here (this code is dead):
https://github.com/sindresorhus/execa/blob/575d464976c022cd127764e078066ba1e7d4dcdc/lib/errname.js#L2

I'm not sure how it arrived, although there is a stale copy here:
https://github.com/octokit/graphql-action/blob/5c1fea5240db8062f5a757c9c3d9a200ab82d970/dist/index.js#L3716